contracts/rewards/AutoCompoundingRewards.sol

```diff
@@ -264,10 +258,43 @@ contract AutoCompoundingRewards is IAutoCompoundingRewards, ReentrancyGuardUpgra
             distributionType: distributionType,
             totalRewards: totalRewards,
             startTime: startTime,
-            endTime: endTime
+            endTime: endTime,
+            halfLife: halfLife
         });
     }
 
+    /**
+     * @inheritdoc IAutoCompoundingRewards
+     */
+    function createFlatProgram(
+        Token pool,
+        uint256 totalRewards,
+        uint32 startTime,
+        uint32 endTime
+    ) external {
+        if (startTime >= endTime) {
+            revert InvalidParam();
+        }
+
+        _createProgram(pool, totalRewards, FLAT_DISTRIBUTION, startTime, endTime, 0);
+    }
+
+    /**
+     * @inheritdoc IAutoCompoundingRewards
+     */
+    function createExpProgram(
```

Comment:
It's ExpDecay

Reply:
Done (as part of a conclusive rename of 'ExponentialDecay' to 'ExpDecay').